src/Nethermind/Nethermind.JsonRpc/JsonRpcService.cs

```diff
@@ -90,7 +90,7 @@ private async Task<JsonRpcResponse> ExecuteAsync(JsonRpcRequest request, string
     {
         JsonElement providedParameters = request.Params;
 
-        LogRequest(methodName, providedParameters, method.ExpectedParameters);
+        if (_logger.IsTrace) LogRequest(methodName, providedParameters, method.ExpectedParameters);
```

Comment:
Should be debug IMO

Reply:
What should Debug be showing and what should Trace be doing?

Debug should be more issues that aren't errors; like timeouts, disconnects etc? Then trace more info about business as usual and this fits into that camp?

Question, do you use debug or trace logs?

Reply:
IMO Debug should show up when something meaningful happen but we dont want to show it to the end user. It is something that dev can reasonably have turned on all the time at least in file for bugs that is very hard to replicate. Eg, we want to know when an rpc started and when it ended. Trace is when we dont have a choice.

Reply:
This one is Trace in LogRequest currently; and just moving it out so it doesn't call method and then exist